village/src/components/Navigator.css

```diff
@@ -0,0 +1,28 @@
+.nav {
+  background-color: #000b1fcc;
+  -webkit-backdrop-filter: blur(50px);
+  -moz-filter: blur(50px);
+  -ms-filter: blur(50px);
+  -o-filter: blur(50px);
+  backdrop-filter: blur(50px);
+  position: absolute;
+  position: fixed;
+  z-index: 1000;
+  top: 0;
+  width: 100%;
+  padding: 15px 0;
+}
+
+.links {
+  color: white;
+  text-decoration: none;
+  padding: 0px 50px;
+  text-transform: uppercase;
+  font-weight: 700;
+  transition: 0.2s ease-in-out;
+}
+
+.links:hover {
+  transition: 0.2s ease-in-out;
+  color: rgba(24, 76, 231, 0.953);
```

Comment:
Again, incredible use of the transitions to just make the UI overall so much more interactive and fun.  I loved it.